aviary/validation_cases/benchmark_tests/test_off_design_FwFm.py

```diff
@@ -0,0 +1,232 @@
+import unittest
+import aviary.api as av
+
+from aviary.interface.default_phase_info.height_energy import phase_info_parameterization
+from aviary.variable_info.enums import ProblemType, Verbosity
+from aviary.variable_info.variables import Mission
+
+from openmdao.utils.testing_utils import require_pyoptsparse, use_tempdirs
+from openmdao.utils.assert_utils import assert_near_equal
+
+
+class HeightEnergyTestCase(unittest.TestCase):
+    def setUp(self) -> None:
+        self.sized_mass = 175871.04745399
+        self.sized_range = 3375
+        phase_info = {
```

Comment:
Could we decrease the lines of code here by using an already declared phase_info.py file (i.e. example_phase_info.py)